src/main/java/prefs/Preferences.java

```diff
@@ -15,83 +21,169 @@
  * It cannot be as we need to mock it.
  */
```

Comment:
Update header comment. As this class is going to be the Preferences API for the rest of the system, it needs to be well documented.